boards/riscv/hifive1/support/openocd.cfg

```diff
@@ -1,7 +1,39 @@
-source [find board/sifive-hifive1.cfg]
```

Comment:
Could you please elaborate why we can't use the config file provided by OpenOCD?

Reply:
The cfg file provided by OpenOCD is not just only configuration. It runs the init function.
The west's OpenOCD runner also does init.
It is not preferable behavior init run twice. (It seems to make it a bit unstable.)

Reply:
ack